ci-operator/config/tnozicka/openshift-acme/master.json

```diff
@@ -0,0 +1,77 @@
+{
+  "base_images": {
+    "os": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "centos",
+      "tag": "7"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.10"
+  },
+
+  "canonical_go_repository": "github.com/tnozicka/openshift-acme",
+
+  "binary_build_commands": "make build GOFLAGS='-v'",
+  "test_binary_build_commands": "make build GOFLAGS='-v -race'",
+
+  "images": [
+    {
+      "from": "os",
+      "to": "openshift-acme",
```

Comment:
@stevekuznetsov I though I am creating the image there but the dependency resolution is still complaining:
```
./ci-operator --config /home/tnozicka/go/src/github.com/openshift/release/ci-operator/config/tnozicka/openshift-acme/master.json --git-ref=tnozicka/openshift-acme@test-ci
2018/08/28 13:26:43 Resolved tnozicka/openshift-acme@test-ci to commit 11fba481be09b7ada4aab2771054d2a3e3c7c68e
2018/08/28 13:26:43 Resolved source https://github.com/tnozicka/openshift-acme to test-ci@11fba481
2018/08/28 13:26:43 Resolving inputs for the test
2018/08/28 13:26:43 Resolved https://api.ci.openshift.org/openshift/release:golang-1.10 to sha256:23c49017c40e1b7c910e3fcf6370ffb31faf5c376402681cbe7379ec949cd5be
2018/08/28 13:26:44 Resolved https://api.ci.openshift.org/openshift/centos:7 to sha256:fc2476ccae2a5186313f2d1dadb4a969d6d2d4c6b23fa98b6c7b0a1faad67685
2018/08/28 13:26:44 Resolved inputs, targetting namespace ci-op-xmbxjnvw
2018/08/28 13:26:44 step <*steps.imagesReadyStep> is missing dependencies: <&api.externalImageLink{image:api.ImageStreamTagReference{Cluster:"", Namespace:"", Name:"openshift-acme", Tag:"ci", As:""}}>
2018/08/28 13:26:44 step <*steps.outputImageTagStep> is missing dependencies: 
2018/08/28 13:26:44 step <*steps.outputImageTagStep> is missing dependencies: <&api.releaseImagesLink{}>
2018/08/28 13:26:44 step <*steps.imagesReadyStep> is missing dependencies: <&api.externalImageLink{image:api.ImageStreamTagReference{Cluster:"", Namespace:"", Name:"openshift-acme", Tag:"ci", As:""}}>
2018/08/28 13:26:44 step <*steps.imagesReadyStep> is missing dependencies: <&api.externalImageLink{image:api.ImageStreamTagReference{Cluster:"", Namespace:"", Name:"openshift-acme", Tag:"ci", As:""}}>
2018/08/28 13:26:44 step <*steps.outputImageTagStep> is missing dependencies: 
2018/08/28 13:26:44 step <*steps.outputImageTagStep> is missing dependencies: <&api.releaseImagesLink{}>
2018/08/28 13:26:44 step <*steps.imagesReadyStep> is missing dependencies: <&api.externalImageLink{image:api.ImageStreamTagReference{Cluster:"", Namespace:"", Name:"openshift-acme", Tag:"ci", As:""}}>
2018/08/28 13:26:44 step <*steps.imagesReadyStep> is missing dependencies: <&api.externalImageLink{image:api.ImageStreamTagReference{Cluster:"", Namespace:"", Name:"openshift-acme", Tag:"ci", As:""}}>
2018/08/28 13:26:44 step <*steps.imagesReadyStep> is missing dependencies: <&api.externalImageLink{image:api.ImageStreamTagReference{Cluster:"", Namespace:"", Name:"openshift-acme", Tag:"ci", As:""}}>
2018/08/28 13:26:44 step <*steps.outputImageTagStep> is missing dependencies: 
2018/08/28 13:26:44 step <*steps.outputImageTagStep> is missing dependencies: <&api.releaseImagesLink{}>
2018/08/28 13:26:44 step <*steps.imagesReadyStep> is missing dependencies: <&api.externalImageLink{image:api.ImageStreamTagReference{Cluster:"", Namespace:"", Name:"openshift-acme", Tag:"ci", As:""}}>
2018/08/28 13:26:44 step <*steps.imagesReadyStep> is missing dependencies: <&api.externalImageLink{image:api.ImageStreamTagReference{Cluster:"", Namespace:"", Name:"openshift-acme", Tag:"ci", As:""}}>
2018/08/28 13:26:44 step <*steps.outputImageTagStep> is missing dependencies: 
2018/08/28 13:26:44 step <*steps.outputImageTagStep> is missing dependencies: <&api.releaseImagesLink{}>
2018/08/28 13:26:44 Ran for 0s
error: could not print execution order: could not sort nodes: steps are missing dependencies
```
any idea what I got wrong?

Reply:
I tested the configuration file in the PR with ci-operator built from master (`24aa5c7`) and had no dependency errors, are you sure it is the same file?

Reply:
Hm, updating the ci-operator sources to HEAD got me past this error.